apps/etterlatte-beregning/src/main/kotlin/sanksjon/SanksjonService.kt

```diff
@@ -44,6 +44,11 @@ class SanksjonService(
         logger.info("Oppdaterer sanksjon med behandlingID=$behandlingId")
         return sanksjonRepository.oppdaterSanksjon(sanksjon, brukerTokenInfo.ident())
     }
+
+    fun slettSanksjon(sanksjonId: UUID) {
+        logger.info("Sletter sanksjon med sanksjonID=$sanksjonId")
```

Comment:
Denne metoden trenger nok en sjekk mot behandling for å se om den kan endres (status typ). Blir vel litt tilsvarende som andre servicer gjør i beregnings-appen

Reply:
Har lagt til sjekk på både slettSanksjon og opprettEllerOppdaterSanksjon